contrib/ovirt/lib/ovirtlago/repoverify.py

```diff
@@ -107,7 +107,13 @@ def fetch_xml(url):
     Returns:
         lxml.etree._Element: Root of the xml tree for the retrieved resource
     """
-    content = urllib2.urlopen(url).read()
+    try:
+        content = urllib2.urlopen(url).read()
+    except urllib2.URLError, e:
+        raise RuntimeError(
+            'Failed to open URL %s\nError code: %s\nError reason: %s' %
+            (url, e.code, e.reason)
```

Comment:
a urllib2.URLError instance has no code member:
  https://docs.python.org/2/library/urllib2.html#urllib2.URLError
But the HTTPError might:
  https://docs.python.org/2/library/urllib2.html#urllib2.HTTPError